### DISCUSSION TITLE
When we click on the time field in the calender, it is considered as outside click

### DISCUSSION THREAD

**Discussion by Lingareddyvinitha** [adobe/react-spectrum#4246] [Q&A]
We used the  OutsideClickHandler from the package `react-outside-click-handler`
when we try to give inputs for the time field in the date picker outside click is triggering

Sample Code
```
<Provider theme={spectrumTheme} colorScheme='light' height='100%'>
 <OutsideClickHandler
        onOutsideClick={() => {
           console.log('outside click triggered')
         }}
  >
    <DatePicker label='Event date' granularity='second' />
 </OutsideClickHandler>
 </Provider>

**Comment by LFDanLu** [comment]
I'm unsure how this `OutsideClickHandler` works but the Calendar rendered by the DatePicker is in a Portal. `react-outside-click-handler` calls out Portals as something it doesn't [handle](https://github.com/airbnb/react-outside-click-handler#children-proptypesnodeisrequired):   
`Note that if you use a Portal (native or react-portal) of any sort in the children, the OutsideClickHandler will not behave as expected.`

What use case are you trying to accomplish with this outside click handler?
